PIDPlanner:
  # next point distance
  p_window: 0.5

  # goal reached tolerance
  goal_dist_tolerance: 0.2
  rotate_tolerance: 0.5

  # linear velocity
  max_v: 2
  min_v: 0.0
  max_v_inc: 0.2
  #前瞻点
  math_path_proportion: 1.09
  # angular velocity
  #max_w: 1.57
  max_w: 0.6
  min_w: 0.0
  max_w_inc: 1.57

  # pid controller params
  k_v_p: 1.00
  k_v_i: 0.10
  k_v_d: 0.10

  k_w_p: 1.00
  k_w_i: 0.10
  k_w_d: 0.12

  k_theta: 0.7

  goal_radius: 1.0
  Lfw: 3.0
  controller_frequency: 20
  wheelbase: 0.3